website/content/en/docs/concepts/policies.md

```diff
@@ -1,37 +1,111 @@
 ---
-title: Policies
+title: Access Policies
 description: Controlling Service access across peers
 weight: 38
 ---
 
-<!--
-Each task should give the user
+Access policies allow users and administrators a fine-grained control over which client workload may access which service.
+This is an important security mechanism for applying [micro-segmentation](https://en.wikipedia.org/wiki/Microsegmentation_(network_security)),
+which is a basic requirement of [zero-trust](https://en.wikipedia.org/wiki/Zero_trust_security_model) systems.
+Access policies can also be used for enforcing corporate security rules, as well as segmenting the fabric into trust zones.
 
-* The prerequisites for this task, if any (this can be specified at the top of a multi-task page if they're the same for all the page's tasks. "All these tasks assume that you understand....and that you have already....").
-* What this task accomplishes.
-* Instructions for the task. If it involves editing a file, running a command, or writing code, provide code-formatted example snippets to show the user what to do! If there are multiple steps, provide them as a numbered list.
-* If appropriate, links to related concept, tutorial, or example pages.
--->
+ClusterLink's access policies are based on attributes that are attached to [peers]({{< ref "peers" >}}),
+[services]({{< ref "services" >}}) and client workloads.
+Each attribute is a key:value pair, similar to how [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) are used in Kubernetes.
+This approach, called ABAC (Attribute Based Access Control), allows referring to a set of entities in a single policy,
+rather than listing individual entity names. Using attributes is safer, more resilient to changes, and easier to control and audit. At the moment, a limited set of attributes is available to use. We plan to enrich this set in the future.
 
-## What is it
+Every instance of an access policy either allows or denies a given set of connections.
+This set is defined by specifying the sources and destinations of these connections.
+Sources are defined in terms of the attributes attached to the client workloads.
+Destinations are defined in terms of the attributes attached to the target services.
+Both client workloads and target services may inherit some attributes from their hosting peer.
 
-### Policy Concepts
+There are two tiers of access policies in ClusterLink. The high-priority tier is intended for cluster/Peer administrators 
```

Comment:
nit: is the dash between high and priority needed?

Reply:
Yes. See [here](https://www.grammarly.com/blog/hyphen-in-compounds-with-high-low/)